components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/utils/APIUtil.java

```diff
@@ -928,6 +922,11 @@ public static String getSwagger12DefinitionFilePath(String apiName, String apiVe
     	return resourcePath;
     }
 
+    public static String getSwagger20DefinitionFilePath(String apiName, String apiVersion, String apiProvider) {
+        return APIConstants.API_DOC_LOCATION + RegistryConstants.PATH_SEPARATOR +
+                apiName +"-"  + apiVersion + "-" + apiProvider + RegistryConstants.PATH_SEPARATOR;
```

Comment:
Fixed in the commit b48cbb0e9a20d8f972a7323ce485823f6541fd26
Thanks for the comments.